server/api/api/endpoints/jobs.py

```diff
@@ -52,6 +55,32 @@ def deploy_monitoring_batch_job(
     :return: model monitoring batch job as a dictionary.
     """
 
+    await server.api.utils.auth.verifier.AuthVerifier().query_project_resource_permissions(
+        resource_type=mlrun.common.schemas.AuthorizationResourceTypes.function,
+        project_name=project,
+        resource_name=mlrun.common.schemas.model_monitoring.MonitoringFunctionNames.BATCH,
+        action=mlrun.common.schemas.AuthorizationAction.store,
+        auth_info=auth_info,
+    )
+
+    if (
```

Comment:
If you know here if the request uses schedule or not. You can reroute the request only if it has a schedule.

Reply:
good note, fixed